.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,176 @@
+name: integration-tests
+on:
+  push:
+    branches: [ "518-implement-semi-automatic-integration-tests" ] #REMOVE WHEN ON MASTER 
+  pull_request_review:
+    types: [submitted]
+  workflow_dispatch:
+    inputs:
+      bob_nodes_amount:
+        description: 'Amount of nodes in bob cluster for tests'
+        required: true
+        default: '4'
+      bob_vdisk_count:
+        description: 'Vdisks count'
+        required: true
+        default: '4'
+      bob_replicas_amount:
+        description: 'Amount of vdisks replicas'
+        required: true
+        default: '2'
+      bob_payload_size:
+        description: 'Payload size'
+        required: true
+        default: '10240'
+      bob_record_count:
+        description: 'Amount of records to process during test'
+        required: true
+        default: '10000'
+
+jobs:
+  build-binaries:
+    if: github.event.review.state == 'approved' || github.event.type != 'PullRequestReviewEvent' 
+    strategy:
+      matrix:
+        key: [8, 16]
+    env:
+      TARGET: "x86_64-unknown-linux-musl"
+      BUILD_MODE: "release"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checks-out repository 
+        uses: actions/checkout@v2
+      - name: Set env variables and install packages
+        run: |
+          echo "BOB_COMMIT_HASH=$(echo ${GITHUB_SHA} | cut -c1-8)" >> $GITHUB_ENV
+          echo "BOB_KEY_SIZE=${{ matrix.key }}" >> $GITHUB_ENV
+          sudo apt-get update && sudo apt-get install -y --no-install-recommends musl-tools
+      - name: Install latest stable
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          target: ${{ env.TARGET }}
+          default: true
+      - name: Build binaries
+        uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --release --target=${{ env.TARGET }}
+      - name: Upload artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: bob-binaries-${{ matrix.key }}
+          path: |
+            target/${{ env.TARGET }}/${{ env.BUILD_MODE }}/bobd
+            target/${{ env.TARGET }}/${{ env.BUILD_MODE }}/bobp
+            target/${{ env.TARGET }}/${{ env.BUILD_MODE }}/ccg
+            target/${{ env.TARGET }}/${{ env.BUILD_MODE }}/brt
+
+  build-docker-images:
+    if: github.event.review.state == 'approved' || github.event.type != 'PullRequestReviewEvent'
```

Comment:
Test runs on every comment to this PR. This should be fixed. Please recheck, that you write the correct event type